hs_core/views/__init__.py

```diff
@@ -668,6 +997,37 @@ def get_metadata_terms_page(request, *args, **kwargs):
     return render(request, 'pages/metadata_terms.html')
 
 
+def _send_email_on_group_membership_acceptance(membership_request):
+    """
+    Sends email notification of group membership acceptance
+
+    :param membership_request: an instance of GroupMembershipRequest class
+    :return:
+    """
+
+    if membership_request.invitation_to is not None:
+        # here we are sending email to group owner who invited
+        email_msg = """Dear {}
+        <p>Your invitation to user '{}' to join the group '{}' has been accepted.</p>
+        <p>Thank you</p>
+        <p>The HydroShare Team</p>
+        """.format(membership_request.request_from.first_name,
+                   membership_request.invitation_to.first_name, membership_request.group_to_join.name)
+    else:
+        # here wre are sending email to the user whose request to join got accepted
+        email_msg = """Dear {}
+        <p>Your request to join the group '{}' has been accepted.</p>
+        <p>Thank you</p>
+        <p>The HydroShare Team</p>
+        """.format(membership_request.request_from.first_name, membership_request.group_to_join.name)
+
+    send_mail(subject="HydroShare group membership",
+              message=email_msg,
+              html_message=email_msg,
+              from_email=settings.DEFAULT_FROM_EMAIL,
+              recipient_list=[membership_request.request_from.email], fail_silently=True)
```

Comment:
@pkdash I remember I suggested you to change fail_silently to True in that case since it was for changing email address, and the original email address may not be a valid one in which case we should make this send_mail call to fail silently without raising exceptions since it is a valid case which might be the reason why they change email address in the first place. In this case, though, the user is requesting to join a group, and the email address has to be valid, and if not and send_mail() call failed, it is probably a good thing for us to know about rather than making it to fail silently.